maven-core/src/main/java/org/apache/maven/execution/MavenExecutionRequest.java

```diff
@@ -442,4 +442,10 @@
      * @since 3.3.0
      */
     Map<String, Object> getData();
+    
+    /**
+     * @return a copy of this
+     */
```

Comment:
Please add a since tag.

Reply:
I thought about that... But which version might it be? ;-)

Reply:
3.7.0

Reply:
Shall I rebase my commits with JIRA associating comments and provide a new pull request?

Reply:
Not necessary, just force push to this one.

Reply:
Which project in JIRA issue? I am confused of that many projects available...

Reply:
https://issues.apache.org/jira/projects/MNG